portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/moresites.vm

```diff
@@ -85,9 +85,9 @@
                                     #end
                                 </a>
                             </div>
-                            <a href="#" id="${site.siteId}" class="toolMenus" aria-haspopup="true" aria-label="$rloader.getFormattedMessage("sit_open_menu", $site.siteTitleTrunc)">
+                            <button id="${site.siteId}" class="toolMenus only-icon-btn" aria-haspopup="true" aria-label="$rloader.getFormattedMessage("sit_open_menu", $site.siteTitleTrunc)">
```

Comment:
This seems to be causing some issues as the id attribute is not unique .... Found via aXe dev tools in Chrome

Reply:
I haven't found the element whose id is the same as the `toolMenus` button. Which is the _Issue Description_?

Reply:
<a class="getSiteDesc" href="[#](http://127.0.0.1:9080/portal/site/cab07ac1-4bfd-4f0e-ae97-4f67c9c43419/tool/e87ca2ba-f636-46f8-a3b2-dfb44a5bef28#)" id="cab07ac1-4bfd-4f0e-ae97-4f67c9c43419">(More)</a>

<button id="cab07ac1-4bfd-4f0e-ae97-4f67c9c43419" class="toolMenus only-icon-btn" aria-haspopup="true" aria-label="Open attached menu for SMPL101 Spring 2022 to access its tools">

Reply:
So the element with the same id belongs to the "(more)" anchor on Site info tool, outside the modal. Should I modify the "toolMenus" button id anyway? ( and therefore the logic behind )

Reply:
Yes, I think we should do whatever it takes to satisfy aXe DevTools. Every a11y review is going to use this tool ....